Blazor/Utils/CDTSMain.cs

```diff
@@ -0,0 +1,19 @@
+﻿using GoC.WebTemplate.Components.Entities;
+using GoC.WebTemplate.Components.Utils;
+using Microsoft.JSInterop;
+
+namespace GoC.WebTemplate.Blazor.Utils
+{
+    public static class CDTSMain
+    {
+        public static async Task InstallCDTS(IJSRuntime js, string lang, SetupBase setupBase = null, bool isApp = false)
+        {
+            await js.InvokeVoidAsync("cdtsBlazor.installCDTS", lang);
+
+            setupBase ??= new SetupBase();
+            setupBase.ExitSecureSite ??= new ExitSecureSite();
+
+            await js.InvokeVoidAsync("cdtsBlazor.setRefTop", JsonPlainSerializationHelper.SerializeToJson(setupBase).ToString(), setupBase.ExitSecureSite, isApp);
```

Comment:
3 main things here:
- It is valid as far as CDTS is concerned to have `null` for `ExitSecureSite` so we should probably keep the same line of thought and leave `ExitsSecurreSite` to `null` if null. (if course, the code in cdts-blazor.js will have to be adjusted to not crash if null ... )
- Since `ExitSecuritySite` is part of `setupBase`, there is no need to pass it as a separate parameter to `cdtsBlazor.setRefTop`. `setRefTop` can take it from setupBase
- At line 16 we're using `JsonPlainSerializationHelper.SerializeToJson` to pass `setupBase` as a JSON string, but we pass `setupBase.ExitSecureSite` directly.  
  - **Does this mean we don't need to serialize setupBase?**
  - **If yes, does this mean we don't have to serialize anything to string before calling all  the `cdtsBlazor.*` functions?**